src/github.com/matrix-org/dendrite/syncapi/storage/output_room_events_table.go

```diff
@@ -44,22 +46,24 @@ CREATE TABLE IF NOT EXISTS syncapi_output_room_events (
     -- A list of event IDs which represent a delta of added/removed room state. This can be NULL
     -- if there is no delta.
     add_state_ids TEXT[],
-    remove_state_ids TEXT[]
+	remove_state_ids TEXT[],
```

Comment:
what's happened with the indentation here?

Reply:
... turns out the rest of the SQL using spaces...